cms/static/js/certificates/views/certificate_details.js

```diff
@@ -7,9 +7,10 @@ define([ // jshint ignore:line
     'gettext',
     'js/views/baseview',
     'js/certificates/models/signatory',
-    'js/certificates/views/signatory_details'
+    'js/certificates/views/signatory_details',
+    'js/views/utils/view_utils'
```

Comment:
I take it that view_utils.js already exists in the code base via another PR, right?